config.xml

```diff
@@ -3,10 +3,10 @@
     <name>Emby</name>
     <description>
     Emby Mobile
-  </description>
+    </description>
     <author email="apps@emby.media" href="http://emby.media">
```

Comment:
I guess this has to be changed to our owl mailing list... and URL should be changed as well.

Reply:
I had inquired about the email address in jellyfin-dev on Riot and there was talk about adding an `apps@` mailing list for Jellyfin. I don't think that has happened yet though.

Reply:
@thornbill `apps@jellyfin.org` now exists!